### ISSUE TITLE
Enhancement: add drain structure type with support in quick forms

### ISSUE THREAD

**Issue by aislinnpearson** [Rothamsted-Ecoinformatics/farm_rothamsted#316]
I had a discussion with NW this morning and we realised that there are some assets not displaying even thought the correct location was selected. The assets are drains, which occasionally have management actions associated with them (mowing, spraying, etc) to keep them clear of plants.

These aren't strictly plant assets - they are structural assets - which is why they don't appear in the quick form. We talked about adding them as plant assets, but it didn't seem quite right. Open to discussion on this but my initial thoughts were:

- [ ] Add 'Drain' as a structure type (Paul)
- [ ] Enable the quick forms so that 'Drain' assets are visible in the quick forms (Paul)
- [ ] Add the structure assets to the NWFP (Aislinn)

I was also thinking we should discuss what and how many of these structure types we need. For example I could envisage adding:

- 'Hedge', 'Tree' and 'wild flower strip' as plant types, but then that wonders into commercial crops, which relates to assigning categories to assets (#144) 
- 'Road', 'Gate', 'Power line' and 'Building' might also be useful structure types (although I think building is covered by land type?)

**Comment by paul121** [comment]
To summarize our chat the other week...

> 'Hedge', 'Tree' and 'wild flower strip' as plant types, but then that wonders into commercial crops, which relates to assigning categories to asset

I agree that something like an asset category would be a better way to manage this when you don't want general terms like "hedge" and "tree" in your `plant_type` taxonomy.

> 'Road', 'Gate', 'Power line' and 'Building' might also be useful structure types (although I think building is covered by land type?)

When we chatted about this we realized the "Default structure types" module was not enabled! This module provides a "Building" structure type but none of the others. We discussed that we could add additional types in the future, but there should be some thought & reason before *everything* gets mapped & added to farmOS
